inc/settings.php

```diff
@@ -125,7 +125,7 @@ function siteorigin_north_settings_init(){
 				'sidebar_width' => array(
 					'label'             => __( 'Sidebar Width', 'siteorigin-north' ),
 					'type'              => 'measurement',
-					'live'              => true,
+					'live'              => false,
```

Comment:
@adiraomj I'm good with a sidebar min as we have other settings to remove the sidebar. Let's do that.